pkg/volume/volume_test.go

```diff
@@ -244,14 +245,15 @@ func TestAbnormalDeleteEventHandle(t *testing.T) {
 
 	vl := createFakeVolumeEventListener(testObjects...)
 	ctx := context.TODO()
-	directCSIClient := vl.directCSIClient.DirectV1beta3()
+	volumeInterface := vl.directCSIClient.DirectV1beta3().DirectCSIVolumes()
```

Comment:
We don't need this change as we use unversioned clients only in plugin. We can remove the changes in this file.